### DISCUSSION TITLE
Flarrent - New frontend for Transmission

### DISCUSSION THREAD

**Discussion by FlafyDev** [transmission/transmission#6118] [Show and tell]
I made a frontend for Transmission with Flutter that utilizes the RPC spec. While it doesn't encompass all the features found in other Transmission clients, I implemented all of the features I personally use and made it as good looking as I could. I am, however, open to PRs if anyone wishes to expand its functionality.

As of now, the client is only available on Linux. Eventually I will add a Windows version, which should be relatively easy thanks to Flutter.

There's a video showcase in the Readme.

Repo: https://github.com/flafydev/flarrent

![image](https://github.com/transmission/transmission/assets/44374434/16e41bef-5205-42b0-ad0f-a0b9cb1d809d)

**Comment by GaryElshaw** [comment]
Lovely work!

**Comment by Pentaphon** [comment]
Seems more like a touchscreen app than a desktop app. 

Also, I would go with "FLorrent" over the current name.

**Reply by FlafyDev** [reply]
I don't know if I'll change the name, but why do you think it should be named this way?

**Reply by Pentaphon** [reply]
Flarrent sounds awkward and unrelated to torrents while FLorrent rolls off the tongue better and is obviously more torrent-related sounding while keeping the FL for Flutter.

**Comment by GaryElshaw** [comment]
It's using Flutter, normally used for devices. It's quite lovely to use, my only criticism would be that there isn't any user control for its transparency.

**Reply by FlafyDev** [reply]
Thanks! You can set the background color in the config file. It's ARGB, so you can also change the transparency.
